src/synopsys-action/synopsys-bridge.ts

```diff
@@ -61,8 +67,75 @@ export class SynopsysBridge {
 
     return -1
   }
+
+  async getLatestVersion(): Promise<string> {
+    const versionArray: string[] = await this.getAllAvailableBridgeVersions()
```

Comment:
once latest tag is attached to bridge package from bridge side this code not needed right?

Reply:
Yes